### DISCUSSION TITLE
Unable to transfer in domain

### DISCUSSION THREAD

**Discussion by cylexVEVO** [vercel/community#3648] [Help]
### Question

Hi, I'm trying to transfer a domain from Google Domains over to Vercel, but I keep receiving an error saying that the auth code is incorrect. I've also just transferred another domain over to Vercel, from Google Domains if that matters. Thanks :)

**Comment by amyegan** [comment]
Hey @cylexVEVO. As a sanity check, could you verify that you have entered the correct auth code for this domain with no additional spaces or missing characters? I want to make sure we've rule out the simplest explanation first, since typos are easy to make.

**Comment by cylexVEVO** [comment]
Yes, I am copy and pasting it directly from Google Domains and can verify that there's no missing characters or extra spaces.

**Comment by amyegan** [comment]
Thanks for checking that. I have a couple more ideas. It's possible that the error message isn't entirely accurate about the cause.

[ICANN](https://www.icann.org/) forces domain registrars to wait 60 days between transfers or between a new registration and subsequent transfer. If you try to transfer during that 60 day window, the transfer will fail.

Most domains are locked by default to prevent unauthorized changes. Make sure you have unlocked your domain from your registrar's domain management area. A locked domain could prevent our system from verifying the auth code.

Can you confirm that your domain has been unlocked for transfer and hasn't been newly registered or transferred within the past 60 days? If neither of those seem to be the problem with this domain, please let me know the domain name so I can check with our domain provider for more info on what is going wrong.

**Reply by cylexVEVO** [reply]
Yes, I've had the domain for much longer than 60 days, and have never transferred it prior to this. I have also made sure that the domain is unlocked. The domain name is tempest.band.

**Reply by cylexVEVO** [reply]
Hi, just checking in to see if you have any updates for me? I just tried to transfer the domain again and it's still not working :(
